### ISSUE TITLE
React Bridge Breaks On Navigation

### ISSUE THREAD

**Issue by fweth** [storyblok/storyblok-react#269]
Hello, my issue with the Storyblok React Bridge is that I often have a folder only for content directly at the root (typically named "content") together with another folder called something like "settings" for all global data.  So when I have e.g. "imprint" in "content" then the bridge tries to open the path `content/imprint` while the real path is just `imprint`.  I know that you can manually set the real path, but it would be nice if you could set some folder as the root for all paths.

The other thing is, even when you manually navigate in the page preview to the correct page, the react bridge seems to break (in Next.js) after a navigation event.  So it only works on the page which the preview opens automatically when starting to edit a page.

**Comment by fgiuliani** [comment]
Hi @fweth!

About the first issue, I'm not sure I'm following. Inside Storyblok, the stories get their slug depending on the folder structure defined, unless you manually set a real path for them. Are you talking about the Storyblok application or the code of your Next.js project?
You can also change the slug of a certain folder, in case you want to.

About the second issue, are you browsing the stories through the right-hand content browser? Or do you mean clicking on the links that you have in the real-time preview inside the visual editor? 
If it's the second option, that's an event difficult to track inside the Visual Editor, and it's not the desired way of browsing through different stories/pages. 
If it's the first, it should work correctly. Could you share a Loom video recording showing the error, in case you are talking about the first scenario?

**Comment by fweth** [comment]
Hello @fgiuliani, thanks for the answer!  Changing the folder slug sounds like what I needed.  I programmed my Next.js project so that e.g. the story "content/imprint" maps to the page "/imprint".  The preview panel tries to load "/content/imprint", which makes sense (because it doesn't know how I handle the data in the Next.js project), but as I said, it would be cool if I could change that once and for all (not manually setting a real path for every single story).

The second issue, yes, I meant browsing stories in the real-time preview.  But if what I described in the first issue can be made to work, I don't really need it anymore.
